docs/guides/monitor/kubernetes-k8s-netdata.md

```diff
@@ -18,261 +31,223 @@ customization, or integration with your preferred alerting methods.
+running a [RabbitMQ](https://learn.netdata.cloud/docs/agent/collectors/go.d.plugin/modules/rabbitmq) workload.
 
-The same goes for metrics coming from the CockroachDB pod running on this same node.
+Click the **Metrics** tab to see real-time metrics from that container. Unsurprisingly, it shows a spike in CPU
+utilization at regular intervals.
 
-![CockroachDB metrics collected via service
-discovery](https://user-images.githubusercontent.com/1153921/85444316-0e925d00-b547-11ea-83ba-b834275cb419.png)
+![Viewing real-time container
+metrics](https://user-images.githubusercontent.com/1153921/109050482-aa580800-7696-11eb-9e3e-d3bdf0f3eff7.png)
 
-Service discovery helps you monitor the health of specific applications running on your Kubernetes cluster, which in
-turn gives you a complete resource when troubleshooting your infrastructure's health and performance.
+### Time-series charts
 
-### Kubelet
+Beneath the health map is a variety of time-series charts that help you visualize resource utilization over time, which
+is useful for targeted troubleshooting.
 
-Let's head back up the menu to the **k8s kubelet** section. Kubelet is an agent that runs on every node in a cluster. It
-receives a set of PodSpecs from the Kubernetes Control Plane and ensures the pods described there are both running and
-healthy. Think of it as a manager for the various pods on that node.
+The default is to display metrics grouped by the `k8s_namespace` label, which shows resource utilization based on your
+different namespaces.
 
-Monitoring each node's Kubelet can be invaluable when diagnosing issues with your Kubernetes cluster. For example, you
-can see when the volume of running containers/pods has dropped.
+![Time-series Kubernetes monitoring in Netdata
+Cloud](https://user-images.githubusercontent.com/1153921/109075210-126a1680-76b6-11eb-918d-5acdcdac152d.png)
 
-![Charts showing pod and container removal during a scale
-down](https://user-images.githubusercontent.com/1153921/85598613-9ab48b00-b600-11ea-827e-d9ec7779e2d4.png)
+Each composite chart has a [definition bar](https://learn.netdata.cloud/docs/cloud/visualize/overview#definition-bar)
+for complete customization. For example, grouping the top chart by `k8s_container_name` reveals new information.
 
-This drop might signal a fault or crash in a particular Kubernetes service or deployment (see `kubectl get services` or
-`kubectl get deployments` for more details). If the number of pods increases, it may be because of something more
-benign, like another member of your team scaling up a service with `kubectl scale`.
+![Changing time-series charts](https://user-images.githubusercontent.com/1153921/109075212-139b4380-76b6-11eb-836f-939482ae55fc.png)
 
-You can also view charts for the Kubelet API server, the volume of runtime/Docker operations by type,
-configuration-related errors, and the actual vs. desired numbers of volumes, plus a lot more.
+## Service metrics
```

Comment:
Applications metrics?

Reply:
Moreover, I remember Manos saying that we are thinking to change "service" to "application" as it's more intuitive to users (relevant to alarm/alerts discussion).

Reply:
[Service](https://kubernetes.io/docs/concepts/services-networking/service/) has a special meaning in k8s, so sounds reasonable.

Reply:
We'll have to engage with Manos on that some more. I don't that we should name something like MySQL an _application_—it's a piece of software that implements an application.

Reply:
Both arguments are fair. The important part is to be aligned.